etc/xeus-cpp/tags.d/xtensor.json

```diff
@@ -0,0 +1,4 @@
+{
+    "url": "https://xtensor.readthedocs.io/en/latest/",
+    "tagfile": "xtensor.tag"
```

Comment:
I suggest we don't install the xtensor tag files as part of this PR, since it should probably be installed by xtensor itself.